docs/README.md

```diff
@@ -364,9 +364,6 @@ We use pytests' [doctest integration](https://docs.pytest.org/doctest.html) to v
 For Transformers, the doctests are run on a daily basis via GitHub Actions as can be 
 seen [here](https://github.com/huggingface/transformers/actions/workflows/doctests.yml).
 
-To include your example in the daily doctests, you need to add the filename that
-contains the example docstring to the [documentation_tests.txt](../utils/documentation_tests.txt).
```

Comment:
There is a new file named `utils/not_doctested.txt`. For a file to be doctested, it **SHOULD NOT** be in that file. We can add a sentence at this place :-)